packages/site-kit/src/lib/components/HoverMenu.svelte

```diff
@@ -10,32 +10,25 @@
 
 <style>
 	.hover-menu {
+		padding: 0.5rem;
+
 		:global {
 			a,
 			button {
-				color: var(--sk-text-2);
-				padding: 1rem 1.3rem;
+				color: var(--sk-fg-2);
+				padding: 1rem;
 				display: block;
 				font: var(--sk-font-ui-medium);
 				text-decoration: none;
 				line-height: 1;
 				width: 100%;
 				text-align: left;
-
-				&:first-child {
-					padding-top: 1.3rem;
-					border-radius: var(--sk-border-radius) var(--sk-border-radius) 0 0;
-				}
-
-				&:last-child {
-					padding-bottom: 1.3rem;
-					border-radius: 0 0 var(--sk-border-radius) var(--sk-border-radius);
-				}
+				border-radius: var(--sk-border-radius-inner);
```

Comment:
These changes make it look like this, which is worse IMO 
![image](https://github.com/user-attachments/assets/88830b2d-44f0-44e4-af95-6e2c75cacc23)

Reply:
Interesting... I was driven to make this change by noticing how the focus ring looks when you use keyboard navigation — this to me looks very silly indeed:

<img width="106" alt="image" src="https://github.com/user-attachments/assets/90fdae2b-a938-41f0-b43a-979c489f4cd4">

<img width="107" alt="image" src="https://github.com/user-attachments/assets/b9323267-adfd-483a-8fd6-534ad4e3ee79">

<img width="104" alt="image" src="https://github.com/user-attachments/assets/93fff939-b7ee-4e0e-aaaa-a7a23f822930">

Personally I think it looks better. Even without the focus ring, on `main` we use weird padding tricks that result in the top and bottom items being larger when they're hovered. The version in this PR more closely reflects the way things work in macOS, which is the authority I trust on these sorts of questions more than any other OS:

<img width="234" alt="image" src="https://github.com/user-attachments/assets/390e09f7-325b-470f-9d0f-9437cd592e0e">